src/foam/nanos/medusa/CheckWebAgent.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package foam.nanos.medusa;
+
+import foam.core.X;
+import foam.nanos.http.WebAgent;
+import javax.servlet.http.HttpServletResponse;
+import java.io.PrintWriter;
+
+/**
+   Health check web agent, intented to be called by load balancers
+   to add/remove targets from target group.
+   Responses from haproxy.org
+ */
+public class CheckWebAgent
```

Comment:
Perhaps a more descriptive name would be helpful. 'Check' is kind of vague.

Reply:
Renamed to HealthCheckWebAgent.